plasmapy/formulary/relativity.py

```diff
@@ -155,3 +155,25 @@ def relativistic_energy(m: u.kg, v: u.m / u.s) -> u.Joule:
     γ = Lorentz_factor(v)
     E = γ * m * c ** 2
     return E
+
+
+@validate_quantities(value={"can_be_negative": False})
+def relativistic_speed(
+    value: [u.J, u.kg * u.m / u.s],
+    particle: ParticleLike,
+    is_kinetic_energy: bool = False,
+) -> u.m / u.s:
+    if value.units.is_equivalent(u.J):
+        if is_kinetic_energy:
```

Comment:
I'm wondering if here we could use the `physical_type` attribute of units so that we wouldn't need the `is_kinetic_energy` keyword.

```Python
if value.unit.physical_type == "energy":
```